tests/benchdnn/graph/input_displacer.cpp

```diff
@@ -536,19 +611,50 @@ int partition_data_displacer_t::gen_quantize_filling(
         }
     }
 
-    ::std::unordered_set<size_t> empty_set;
-    const auto &eng = get_test_engine();
-    ref_primitive_t ref_prim(op);
-    ref_prim.init_prb(res);
-    if (res->state == INVALID_ARGUMENTS) return FAIL;
```

Comment:
Why has such (actually, any kind of) handling disappeared from the new branch of the code?
Is it impossible to get any errors in there?